src/main/java/Duke.java

```diff
@@ -1,10 +1,39 @@
+import java.util.Scanner;
+
 public class Duke {
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+        UI.showWelcomeScreen();
+
+        Scanner sc = new Scanner(System.in);
+
+        Task[] tasks = new Task[100];
```

Comment:
Perhaps you might want to first declare the array size as a constant, then use the constant to initialize your array so as to improve your code readability and thus the quality.

Reply:
Indeed, good point!